src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,11 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
+    private final Random random = new Random();
```

Comment:
you do not use this variable in methods. you also need to separate class fields and methods with an empty line. Likewise in other cases